source/stylesheets/about.css.scss

```diff
@@ -237,7 +254,7 @@ body.index.responsive #content {
       margin: 0 auto;
       padding-top: 40px;
     }
-	a:hover {
+  a:hover {
     border-bottom: none;
   }
-}
+}
```

Comment:
is the `.emberconf` properly formatted (indentation)? can't tell if it's the diff or the code